Makefile

```diff
@@ -94,12 +95,12 @@ $(BIN)/license-header: $(BIN) Makefile
 
 $(BIN)/golangci-lint: $(BIN) Makefile
 	GOBIN=$(abspath $(@D)) $(GO) install \
-		github.com/golangci/golangci-lint/cmd/golangci-lint@v1.56.1
+		github.com/golangci/golangci-lint/cmd/golangci-lint@$(GOLANGCI_LINT_VERSION)
 
 $(BIN)/protovalidate-conformance: $(BIN) Makefile
 	GOBIN=$(abspath $(BIN)) $(GO) install \
     	github.com/bufbuild/protovalidate/tools/protovalidate-conformance@$(CONFORMANCE_VERSION)
 
-$(BIN)/protovalidate-conformance-go: $(BIN) Makefile
-	GOBIN=$(abspath $(BIN)) $(GO) build -o $(BIN)/protovalidate-conformance-go \
-		./internal/cmd/protovalidate-conformance-go
+.PHONY: protovalidate-conformance-go
```

Comment:
Just made this a PHONY target - the go compiler is super fast and this will ensure we always keep the binary updated when running targets like `make conformance`.